src/Updater.php

```diff
@@ -327,6 +327,13 @@ protected function newVersionAvailable()
 
     protected function backupPhar()
     {
+        $tempDirectory = $this->getTempDirectory();
```

Comment:
This block is copy-pasted several times. Maybe it can be moved out to separate method?